content/week-two/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week Two
+content_type: topic
+submission type:  
+---
+
+# Challenge: Improve working professionals' experiences of working remotely

```

Comment:
If the "title" says good and useful things then this extra title line wouldn't be needed